Azure/azure-ad/ingest/parser.yml

```diff
@@ -79,6 +79,8 @@ stages:
           azuread.properties.deviceDetail.trustType: "{{parsed_event.message.properties.deviceDetail.trustType}}"
           azuread.properties.original_transfer_method: "{{parsed_event.message.properties.originalTransferMethod}}"
           azuread.properties.session_id: "{{parsed_event.message.properties.sessionId}}"
+          azuread.properties.network_location.network_type: "{{(parsed_event.message.properties.networkLocationDetails | first).networkType}}"
+          azuread.properties.network_location.network_names: "{{(parsed_event.message.properties.networkLocationDetails | first).networkNames}}"
```

Comment:
Can you add a filter to ensure that `parsed_event.message.properties.networkLocationDetails` has at least one item? In order to avoid `(parsed_event.message.properties.networkLocationDetails | first) = null`